Sources/Elementary/HtmlAttributes.swift

```diff
@@ -123,6 +123,30 @@ public extension HTMLAttribute where Tag == HTMLTag.link {
     }
 }
 
+// button tag attributes
+// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button
+public extension HTMLAttribute where Tag == HTMLTag.button {
+    struct ButtonType: Sendable, ExpressibleByStringLiteral {
+        var value: String
+
+        init(value: String) {
+            self.value = value
+        }
+
+        public init(stringLiteral value: String) {
+            self.value = value
+        }
```

Comment:
I don't think we need to have an expressiblebystringliteral initializer to this one. 
there are only these three valid options.

the InputType also does not have one for the same reason.